SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTestSettings.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialTestSettings);
+
+FSpatialTestSettings::FSpatialTestSettings()
+	: OriginalSpatialGDKSettings(nullptr)
+	, OriginalLevelEditorPlaySettings(nullptr)
+	, OriginalSpatialGDKEditorSettings(nullptr)
+	, OriginalGeneralProjectSettings(nullptr)
+	, OriginalEditorPerformanceSettings(nullptr)
+{
+}
+
+void FSpatialTestSettings::Override(const FString& MapName)
+{
+	// Back up the existing settings so they can be reverted later
+	Duplicate<USpatialGDKSettings>(OriginalSpatialGDKSettings);
```

Comment:
These type names should be redundant and can be removed

Reply:
Good point. Removed.